web-client/src/lib.rs

```diff
@@ -151,10 +162,46 @@ impl ClientConfiguration {
         self.log_level = log_level.to_lowercase();
     }
 
-    /// Instantiates a client from this configuration builder.
-    #[wasm_bindgen(js_name = instantiateClient)]
-    pub async fn instantiate_client(&self) -> Client {
-        Client::create(self).await
+    // /// Instantiates a client from this configuration builder.
+    // #[wasm_bindgen(js_name = instantiateClient)]
+    // pub async fn instantiate_client(&self) -> Client {
+    //     match Client::create(&self.build()).await {
+    //         Ok(client) => client,
+    //         Err(_) => unreachable!(),
+    //     }
+    // }
```

Comment:
Maybe remove the commented code?

Reply:
I am keeping it, but adding a TODO comment that explains that this method should be tried to be reactivated in the future, for its better DX.